share/spice/quixey/quixey.js

```diff
@@ -123,7 +123,7 @@
                     img_m:         screenshot,
                     title:         item.name,
                     heading:       item.name,
-                    rating:        item.rating,
+                    rating:        features.rating_avg || item.rating,
```

Comment:
@marianosimone actually we should try and use `custom.avg_rating` when it's defined. It looked like it was accurate, I'm not sure about the `rating_avg` property.